std_msgs/Header header

ObjectDescription description

string vehicle_type		
string vehicle_role		
string navigation_segment		
string lane_id
string lanes_from_left

bool head_light_state
bool brake_light_state
bool reverse_light_state

string signal_state
string signal_light

WheelMsg[] wheels